icu4c/source/common/dictbe.h

```diff
@@ -362,10 +369,23 @@ class CjkBreakEngine : public DictionaryBreakEngine {
      * @internal
      */
   UnicodeSet                fHangulWordSet;
+  UnicodeSet                fNumberWordSet;
```

Comment:
as I mention in the Java PR, since fNumberWordSet  and fOpenPunctuationSet are use exactly in the same place, we should merge them into one UnicodeSet fNumberOrOpenPunctuationSet w/ u"[[:Nd:][:Pi:][:Ps:]]" pattern

Reply:
Resolved